diff -up qtquick3d-everywhere-src-6.4.0-beta3/src/3rdparty/embree/common/sys/vector.h.1~ qtquick3d-everywhere-src-6.4.0-beta3/src/3rdparty/embree/common/sys/vector.h
--- qtquick3d-everywhere-src-6.4.0-beta3/src/3rdparty/embree/common/sys/vector.h.1~	2022-08-11 08:36:05.000000000 +0200
+++ qtquick3d-everywhere-src-6.4.0-beta3/src/3rdparty/embree/common/sys/vector.h	2022-08-20 22:44:10.504076483 +0200
@@ -134,7 +134,7 @@ namespace embree
       {
         /* destroy elements */
         for (size_t i=0; i<size_active; i++)
-          alloc.destroy(&items[i]);
+          std::allocator_traits<decltype(alloc)>::destroy(alloc, &items[i]);
         
         /* free memory */
         alloc.deallocate(items,size_alloced); 
